west.yml

```
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr-network-info
       path: deps/modules/lib/network-info
-      revision: v1.1.0
+      revision: szczys/better-zcbor-encoding
```

Should this be the release tag?

```suggestion
      revision: v1.1.1
```

---

Yes, apologies for the gaff. :facepalm: